internal/server/middleware_test.go

```diff
@@ -59,30 +59,6 @@ func issueToken(t *testing.T, db data.GormTxn, identityName string, sessionDurat
 	return body
 }
 
-func TestRequestTimeoutError(t *testing.T) {
-	router := gin.New()
-	router.Use(TimeoutMiddleware(100 * time.Millisecond))
-	router.GET("/", func(c *gin.Context) {
-		time.Sleep(110 * time.Millisecond)
-
-		assert.ErrorIs(t, c.Request.Context().Err(), context.DeadlineExceeded)
-
-		c.Status(200)
-	})
-	router.ServeHTTP(httptest.NewRecorder(), httptest.NewRequest("GET", "/", nil))
-}
-
-func TestRequestTimeoutSuccess(t *testing.T) {
```

Comment:
I removed this test because the function it was testing was removed, and the behaviour is covered by basically every API test we have.